config/configtelemetry/configtelemetry.go

```diff
@@ -36,8 +36,6 @@ const (
 	levelDetailedStr = "detailed"
 )
 
-const UseOpenTelemetryForInternalMetrics = false
```

Comment:
We should deprecate this for the moment and make it one of the option for one release to enable this.

Reply:
As in keep the variable so it is not a breaking change for the release?

Reply:
yes, deprecate and keep if for 1 release, and next release we remove it.

Reply:
Should this be used as the default value of `enabled` in the feature gate registration to ensure that changes to its value are still honored in the absence of an explicit feature gate CLI flag while it remains?

Reply:
yes we can.